pil_analyzer/src/pil_analyzer.rs

```diff
@@ -890,6 +900,17 @@ namespace N(65536);
     col fixed A = [0]*;
     col fixed C(i) { (Assembly.A((i + 2)) + 3) };
     col fixed D(i) { Assembly.C((i + 3)) };
+"#;
+        let formatted = process_pil_file_contents::<GoldilocksField>(input).to_string();
+        assert_eq!(formatted, input);
+    }
+
+    #[test]
+    fn if_expr() {
+        let input = r#"namespace Assembly(2);
+    col fixed A = [0]*;
+    col fixed C(i) { if (i < 3) { Assembly.A(i) } else { (i + 9) } };
+    col fixed D(i) { if Assembly.C(i) { 3 } else { 2 } };
```

Comment:
How does this work if C isn't boolean?

Reply:
We don't have a type checker yet, so it is defined in the same way as it is defined in C.